lib/sync/sync-server.js

```diff
@@ -356,6 +356,8 @@ module.exports = {
     listCollisions: listCollisions,
     removeCollision: removeCollision,
     callHandler: callHandler,
-    getStats: getStats
+    getStats: getStats,
+    setRecordHashFn: hashProviderModule.setRecordHashFn,
```

Comment:
Also, this doesn't look related to the goal of the PR. 
We can include it if you want, but would be good to get more context